articles.md

```diff
@@ -76,6 +78,7 @@ tags: [node, wallet, security, privacy, nokyc]
 |----------------------------------|----------|-----------------------------------------------------------------|----------------------------------------|
 | [Peach Bitcoin Mobile P2P Exchange](/peach)   |  Guest post by [Bitlevi](https://twitter.com/Bit_levi) on using Peach Bitcoin  |  29 May 2023         |
 | [RoboSats Lightning P2P Exchange](/robosats)   |  Guide for purchasing P2P sats via Lightning using RoboSats  |  22 May 2023            |
+| [How To Buy Bitcoin On Bisq](https://www.athena-alpha.com/how-to-buy-bitcoin-on-bisq/)   |  Guide by [Athena Alpha](https://twitter.com/athena_alpha_) on using Bisq  |  12 May 2023            |
```

Comment:
There is already a guide for Bisq line 84 below or at bitcoiner.guide/bisq

Reply:
You're right yes sorry we missed that. Our guide is a bit more up to date though than the 2020 one, but understand if it's not included in the PR :)